application/frontend/tests/auth.setup.ts

```diff
@@ -4,6 +4,7 @@ import { firstSuperuser, firstSuperuserPassword } from "./config.ts"
 const authFile = "playwright/.auth/user.json"
 
 setup("authenticate", async ({ page }) => {
+  await page.context().clearCookies();
```

Comment:
Is there a particular reason this is needed? Appears to work with or without.

Reply:
was left over from trying to pause animations will remove